test/integration/helpers/extend.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var _merge = function (target, source) {
+  target = target || {};
+  for (var prop in source) {
+    if (typeof source[prop] === 'object') {
+      target[prop] = _merge(target[prop], source[prop]);
+    } else {
+      target[prop] = source[prop];
+    }
+  }
+  return target;
+};
+
+// merge any number of source objects into a destination target object
+// var newObj = extend({}, obj, otherObj);
+// extend(Foo.prototype, Bar.prototype, Baz.prototype);
+var extend = function() {
+  var args = Array.prototype.slice.call(arguments);
+  var target = args[0] || {};
+  var sources = args.slice(1);
+
+  sources.forEach(function(source) {
+    _merge(target, source);
+  });
+  return target;
+};
```

Comment:
Yeah I think I'd rather use `_.extend` (or even `angular.extend`) rather than rewriting it. I don't really see a reason not to include lodash in our test code if we need it.